src/utils/pst.ts

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
+
+// ArDrive Profit Sharing Community Smart Contract
+const vertoCacheURL =
+  "https://v2.cache.verto.exchange/-8A6RexFkpfWwuyVO98wzSFZh0d6VJuI-buTJvlwOJQ";
+
+// Calls the ArDrive Community Smart Contract to pull the fee
+export async function getArDriveFee(): Promise<number> {
+  try {
+    const res = (await axios.get(vertoCacheURL)).data;
+
+    const arDriveCommunityFee = res.state.settings.find(
+      (setting: (string | number)[]) =>
+        setting[0].toString().toLowerCase() === "fee"
+    );
+    return arDriveCommunityFee ? arDriveCommunityFee[1] : 15;
```

Comment:
Your default here is an integer representation of a percentage, 15, whereas the default a few lines below is a decimal representation (.15).

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)